### PR TITLE
Celery: sane broker URI, README update

### PR DESCRIPTION
**Pull request by justinnoah**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,8 +60,12 @@ Seed the database with the test users (refer to the source code for the user det
 
 # Running the Development Server
 
+<<<<<<< HEAD
 To start up the development server with live reloading of python code changes (`r` flag) and debug output (`d` flag):
 
+=======
+    $ celery -A _15thnight.celery worker
+>>>>>>> Celery: sane broker URI, README addition
```
Comment (nobane) [inline README.md]: Looks like there are some merge conflicts here.

**Comment by justinnoah** [comment]
Rebased

File: README.md [hunk 2]
```diff
@@ -78,7 +78,11 @@ Seed the database with the test users (refer to the source code for the user det
 
 ## Running the Development Server
 
-To start up the development server with live reloading of python code changes (`r` flag) and debug output (`d` flag):
+Begin by running the celery daemon:
+
+    $ celery -A _15thnight.celery worker
+
+and to start up the development server with live reloading of python code changes (`r` flag) and debug output (`d` flag):
```
Comment (nobane) [inline README.md]: Change this text to:

```
Then start up the development web server with live reloading of python code changes (`r` flag) and debug output (`d` flag):
```

**Comment by nobane** [comment]
There needs to be hosting instructions for celery.

**Comment by justinnoah** [comment]
@spectralsun Instructions added as well as a script to do things for them.

The script creates a config file (borrows from your sed magic), creates an init script, and creates a celery user. It's not a robust script, so manual instructions are included in the readme so the admin can follow what's going on and change anything as necessary.

File: README.md [hunk 3]
```diff
@@ -127,6 +131,21 @@ To host the project from a production environment, first follow the instructions
...
+
+    ./setup_celery.sh
+
+### Manually
+
+    # Change the 3 to the concurrency level desired
+    sed \{"s?PROJECT_PATH?$(pwd)?g; s?THREAD_COUNT?3?g;"\} celeryd.template > celeryd.conf
+    mv celeryd.conf /etc/default/celeryd
+    cp celeryd.init /etc/init.d/celeryd
+    rm -f celeryd.partial
```
Comment (nobane) [inline README.md]: Where does this partial file come from?

File: setup_celery.sh [hunk 4]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+if [ $(id -u) != 0 ]
+then
+    echo "This script must run this as root."
+    exit 1
+fi
+
+echo "Seting up Celery"
```
Comment (nobane) [inline setup_celery.sh]: Typo here with "setting"

File: setup_celery.sh [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+CPU_COUNT=$(cat /proc/cpuinfo|grep precessor|wc -l)
+let "THREAD_COUNT = $CPU_COUNT + 1"
```
Comment (nobane) [inline setup_celery.sh:11]: Is `THREAD_COUNT`/`CPU_COUNT` used?

File: celery_app.py [hunk 6]
```diff
@@ -0,0 +1 @@
+from _15thnight import celery
```
Comment (nobane) [inline celery_app.py:1]: Is this file ever used?

Comment (justinnoah) [inline celery_app.py:1]: yes, that is the entry point for celery (later used by slack)

**Comment by nobane** [comment]
+1

**Pull request merged**